package.json

```diff
@@ -131,6 +131,16 @@
 		"style-loader": "^0.19.1",
 		"webpack-fix-style-only-entries": "^0.4.0"
 	},
+	"resolutions": {
```

Comment:
Nice! I did not know that existed!

Reply:
Sadly, I had to go this route because some nested dependencies were outdated.

Reply:
@olafleur only in yarn :)

Reply:
@EvanHerman : yeah, not always an ideal situation, but it is sometimes a useful tool, like in this case :)